src/components/common/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+// Components
+import { TabItemType } from 'components/OrderBuySell'
+import TabItem from 'components/common/Tabs/TabItem'
+import TabContent from 'components/common/Tabs/TabContent'
+
+interface Props {
+  tabItems: TabItemType[]
+}
+
+const Tabs: React.FC<Props> = (props) => {
+  const [activeTab, setActiveTab] = useState<number>(1)
+
+  const onTabClick = (id: number): void => {
```

Comment:
how is this fn different from the original state setting fn?

Reply:
Good eye :+1:

Reply:
Not sure I follow. You mean why use the useState hook vs. something else, or?

Reply:
what does `onTabClick` do any different than `setActiveTab` ? Look at the way you're using it on line `43`

Reply:
Ah I see now. I could simply pass the setActiveTab directly without having to put it in a separate function :)